### DISCUSSION TITLE
Getting rejected because 13 y/o.

### DISCUSSION THREAD

**Discussion by madebyshaurya** [community/community#134868] [GitHub Education]
### Hi everyone,

I submitted my request for a Education Pack and got rejected because my school was an elementary school. I used my report card for proof and I have been accepted once in 2022. In canada some schools for grade 1-8 are called Elementary but that doesn't mean I'm not thirteen. I will go to grade 8 this year. If someone can help me how to get not rejected please share. Thanks,
Shaurya

**Comment by eliancheng** [comment]
Hi Shaurya,

I understand your frustration with the rejection. 
It might help to reach out directly to GitHub Education support and explain your situation clearly. You can mention that you are currently in grade 8 and provide any additional documentation, like a letter from your school or your current report card, to verify your status. 
Emphasizing that you were accepted previously could also strengthen your case. 
Good luck, and I hope you get the support you need!

**Reply by madebyshaurya** [reply]
How can I contact them?
